src/user/auth/login/login.js

```diff
@@ -42,20 +42,28 @@ class Login extends Component {
     const { loginOptions } = this.state
     return (
       <div className="login-page">
-        <div className="welcome-text">
-          <div className="logo">
-            <DonutTitle />
+        <div className="container-wrapper">
+          <div className="section-home">
+          <div className="welcome-text">
+            <div className="logo">
+              <DonutTitle />
+            </div>
+            <div className="tagline">
+              <h1>One place for meeting everyone.</h1>
+            </div>
+            <div className="description">
+              <p>
+                An Open Source Social networking bridge between Developers,
+                Organisations and Open Source aspirants
+              </p>
+            </div>
+            <div class="login-btn">
+              <a href="#login">
+                Sign In
```

Comment:
find the proper way to do the same cc @ksraj123

Reply:
@ksraj123, Whenever you're free, please let me know about how to proceed with this.

Reply:
using anchor tags for links which do not take you to a different website is not a good practice, for the use case here I think you want the user to be redirected to a different section of the page when he clicks on Sign In, for that you many use [react-router-hash-link](https://www.npmjs.com/package/react-router-hash-link)

Apart from this, Please try to avoid unnecessarily nesting of HTML elements verbose markup should be avoided, provide meaningful names to classes look into BEM naming convention, these suggestions are applicable for not just this piece of code but across the project.

Reply:
Okay, Sure thing! I'll follow this up and push the updated code.
But, I'm having issues in understanding how would I replace all divs with a single one?